lib/strategy/ordered.ts

```diff
@@ -52,7 +66,7 @@ export class OrderedStrategy {
     private pollInterval?: number;
```

Comment:
`toPoll` is again the same as the modulator right?

Reply:
I think not in this case. In the ordered strategy the local `toPoll` property has a `Heap<T>` type, while the modulator's mutable is a `Map<string, T>`. We cannot guarantee the proper ordering with a Map right?